addons/mail/static/src/models/message_tracking_value.js

```diff
@@ -0,0 +1,184 @@
+                 *
+                 * Selection formatter exists but requires knowing all
+                 * possibilities and they are not given in this context.
+                 */
+                case 'char':
+                case 'many2one':
+                case 'selection':
+                    return format.char(value);
+                case 'date':
+                    if (value) {
+                        value = moment.utc(value);
+                    }
+                    return format.date(value);
+                case 'datetime':
+                    if (value) {
+                        value = moment.utc(value);
+                    }
+                    return format.datetime(value);
+                case 'float':
+                    return format.float(value);
+                case 'integer':
+                    return format.integer(value);
+                case 'text':
+                    return format.text(value);
+                case 'monetary':
+                    return format.monetary(value, undefined, {
+                        currency: this.currencyId
+                            ? this.env.session.currencies[this.currencyId]
+                            : undefined,
+                        forceString: true,
+                    });
+                default : 
+                    throw new Error("Message tracking value format is not correctly handled");
+            }
+        },
+    },
+    fields: {
+        /**
+         * Indicate the original field of changed tracking value, such as "Status", "Date".
+         */
+        changedField: attr(),
+        /**
+         * The formatted string of the chanegd field name, such as "Status", "Date", etc.
+         */
+        changedFieldAsString: attr({
+            compute: '_computeChangedFieldAsString',
+        }),
+        /**
+         * Used when the currency changes as the tracking value. This only makes sense for field of type monetary.
+         */
+        currencyId: attr(),
+        /**
+         * Indicate the type of the tracking value. 
+         * The supported types are: boolean, char, many2one, selection, date, datetime, float, integer, text, monetary.
+         */
+        fieldType: attr(),
+        /**
+         * Unique identifier for this tracking value message.
+         */
+        id: attr({
+            required: true,
+            readonly: true,
+        }),
+        /**
+         * The id of the message that the tracking value changes linked to.
+         */
+        message: one('Message', {
+             inverse: 'trackingValues',
+             required: true,
+         }),
+        /**
+         * The new value for the tracking message.
+         */
+        newValue: attr(),
+        /**
+         * The formatted of the new value for the tracking message.
+         */
+        newValueAsString: attr({
+            compute: '_computeNewValueAsString',
+            default: 'None',
```

Comment:
Should be translatable. You cannot provide function to `default` yet I think, so workaround is to fallback `this.env._t("None")` in the compute method.

Reply:
Originally, it is only "None" for everything. But I think you are right, and I will give translate into this field.

Reply:
It was in the template as a text node, so it was automatically registered for translation. In JS it needs to be explicitly wrapped in `_t()`.

Reply:
Nice to know that! Thanks a lot.